### PR TITLE
Fix Visual Studio IntelliSense when specifying the language standard compiler flag

### PR DESCRIPTION
**Pull request by lacasseio**
Fixes https://github.com/gradle/gradle-native/issues/975

### Gradle Core Team Checklist
- [ ] Verify design and implementation 
- [ ] Verify test coverage and [CI build status](https://builds.gradle.org/project.html?projectId=Gradle_Check&tab=projectOverview&branch_Gradle_Check=lacasseio%2Fvisual-studio-2017-intellisense-fix)
- [ ] Verify documentation
- [ ] Recognize contributor in release notes

### CODE REVIEW

**Comment by lacasseio** [comment]
I can confirm opening the VS solution with `LanguageStandard` configuration in VS2015 doesn't give any breaking errors.

File: subprojects/ide-native/src/main/java/org/gradle/ide/visualstudio/internal/AbstractCppBinaryVisualStudioTargetBinary.java [hunk 1]
```diff
@@ -53,6 +53,11 @@ protected AbstractCppBinaryVisualStudioTargetBinary(String projectName, String p
         this.projectLayout = projectLayout;
     }
 
+    @Override
+    public LanguageStandard getLanguageStandard() {
+        return LanguageStandard.from(getBinary().getCompileTask().get().getCompilerArgs().get());
```
[the remark this answers is not in the document]
Comment (lacasseio) [reply subprojects/ide-native/src/main/java/org/gradle/ide/visualstudio/internal/AbstractCppBinaryVisualStudioTargetBinary.java]: As we discuss, this is an internal API. I would only change it to make it more flexible when we implement the C++ language standard.

**Comment by ThadHouse** [comment]
I think this is getting picked up at the wrong time, I have the right arguments set, but this is not getting configured, even on gradle 6.0.1

**Comment by lacasseio** [comment]
@ThadHouse could you open an issue with as much information as possible on what you are seeing? A sample project reproducing the issue would be very beneficial.

**Comment by ThadHouse** [comment]
Yup, I'll work on it. 

Its expected that this would also have fixed old model based vs solutions too, right? Since it seems like they used the same classes.

**Comment by lacasseio** [comment]
Yes, according to the tests, it should fix for both software model and current model plugin.

**Pull request merged**